src/EventListener/AttributesListener.php

```diff
@@ -27,12 +27,23 @@ public function onKernelRequest(RequestEvent $event): void
         $request = $event->getRequest();
         $controller = $this->controllerResolver->getController($request);
 
-        if (!is_array($controller) || 2 !== count($controller)) {
+        if (false === $controller) {
```

Comment:
this seems a bit dangerous to me. what if the `$controller` is something weird, like a different kind of array, or some non-array non-object but not false?

should we not add the is_object condition here? that should also avoid the uncertainty below whether method and class are null or set.

```suggestion
        if (!is_object($controller) && (!is_array($controller) || 2 !== count($controller))) {
```

or we could do an if object, elseif array, else return.

Reply:
Return type for `\Symfony\Component\HttpKernel\Controller\ControllerResolverInterface::getController` is `callable|false`, so return values are limited, but you are right, there can be `string` for example.

I've changed a logic, now it will work only for `object` and `array` with two elements